### ISSUE TITLE
Adjust Twitter window reaction to X

### ISSUE THREAD

**Issue by dreamscached** [Monika-After-Story/MonikaModDev#10211]
https://github.com/Monika-After-Story/MonikaModDev/blob/06baf319a34c2ef585bc7c0a1e969a7eaa894b35/Monika%20After%20Story/game/script-windowreacts.rpy#L267-L282

Nuff said. Twitter is now X and the window reaction has to be adjusted. I would've created a PR myself but I'm concerned if changing the category would require a persistent modifications.

![Screenshot of Monika X profile tab](https://github.com/Monika-After-Story/MonikaModDev/assets/74068927/53f32d50-d570-4e5e-aada-f1edbec58e5a)

**Comment by Booplicate** [comment]
Or we could intentionally keep it `Twitter`

**Comment by Booplicate** [comment]
Jokes aside, I'm a little afraid of porn sites reacts if we keep it as just `/ x`

**Comment by dreamscached** [comment]
Weeeell... yeah. But in case of X/Twitter we could react on stuff that looks exactly like

`Name (@handle) / X`

If possible. That is an issue with X-sites yeah, but if we trim the string, there should just `/ X` be left at the end?

**Comment by TheBearodactyl** [comment]
> https://github.com/Monika-After-Story/MonikaModDev/blob/06baf319a34c2ef585bc7c0a1e969a7eaa894b35/Monika%20After%20Story/game/script-windowreacts.rpy#L267-L282
> 
> Nuff said. Twitter is now X and the window reaction has to be adjusted. I would've created a PR myself but I'm concerned if changing the category would require a persistent modifications.
> 
> ![Screenshot of Monika X profile tab](https://private-user-images.githubusercontent.com/74068927/306752993-53f32d50-d570-4e5e-aada-f1edbec58e5a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkwMDMzNDksIm5iZiI6MTcxOTAwMzA0OSwicGF0aCI6Ii83NDA2ODkyNy8zMDY3NTI5OTMtNTNmMzJkNTAtZDU3MC00ZTVlLWFhZGEtZjFlZGJlYzU4ZTVhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjIxVDIwNTA0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEwNzFiNmEyNjI5NjVmYTFjMTRlNTkxMDA1N2IyZGE0MDU5YWMzZWM1ZTJjZWYzY2IwODAxYTViOTZhZTkxNzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EEUdThMil2PN-8R6XWj3O5jVtMwprQ9EWKbqP_giFVs)

maybe just make it react to `x.com` but have it shown as twitter since nobody calls it "X" anyways

**Comment by dreamscached** [comment]
However, it's 'X' now and I think we should reflect the change. It'll sure take a while for people to move on from it (and they themselves kind of stall at that migration), but it's no longer Twitter really.

**Comment by TheBearodactyl** [comment]
> However, it's 'X' now and I think we should reflect the change. It'll sure take a while for people to move on from it (and they themselves at X kind of stall at that migration), but it's no longer Twitter really.

it is twitter. if elon won't call his daughter by her prefered name, we shouldn't call Twitter X. simple as that

**Comment by dreamscached** [comment]
I don't think Elon's family relations are relevant in the question. The website and the company were renamed to X, officially.

**Comment by TheBearodactyl** [comment]
> I don't think Elon's family relations are relevant in the question. The website and the company were renamed to X, officially.

loophole time :D

https://x.com/racistriamu/status/1686945859320369152

**Comment by TheBearodactyl** [comment]
also
![image_2024-06-21_160319548](https://github.com/Monika-After-Story/MonikaModDev/assets/114454115/3354d2ce-9af7-4f7d-8eef-d35ea77492bc)

**Comment by dreamscached** [comment]
Like I said, I do admit they stall at the migration at the moment. The frontpage however is located at x.com and their company name is 'X'.

[Source 1](https://web.archive.org/web/20230418023557/https://slate.com/technology/2023/04/twitter-inc-x-corp-elon-musk-x-nevada.html)
[Source 2](https://web.archive.org/web/20230412032727/https://storage.courtlistener.com/recap/gov.uscourts.cand.395023/gov.uscourts.cand.395023.123.0.pdf) attention)

**Comment by TheBearodactyl** [comment]
> Like I said, I do admit they stall at the migration at the moment. The frontpage however is located at x.com and their company name is 'X'.
> 
> [Source 1](https://web.archive.org/web/20230418023557/https://slate.com/technology/2023/04/twitter-inc-x-corp-elon-musk-x-nevada.html) [Source 2](https://web.archive.org/web/20230412032727/https://storage.courtlistener.com/recap/gov.uscourts.cand.395023/gov.uscourts.cand.395023.123.0.pdf) (this one deserves attention)

And like i said, who tf cares if elon wants to call it X, it's twitter and it will always be twitter.

**Comment by Booplicate** [comment]
@TheBearodactyl they won't get rid of the old domain because it worth a lot, it'd be a waste and many links would become dead. It's not really a proof. People call it both X and Twitter now.

I still think due to the stupid name we should probably just remove the react. The amount of false positives you get is too high. There's just 3 characters to match.

**Comment by Booplicate** [comment]
> they did get rid of the old domain on the main site but yeah, agreed. maybe replace it with bluesky?

Last time I checked they still owned the domain and it still leads to X.

**Comment by dreamscached** [comment]
I agree with removing the react, to be honest. \@lilmonix3 is *seemingly* (to me) a canon thing endorsed by Team Salvato, but however holds the handle posts rarely and it doesn't quite (again, to me) align with MAS 'canon'.

**Comment by TheBearodactyl** [comment]
> > they did get rid of the old domain on the main site but yeah, agreed. maybe replace it with bluesky?
> 
> Last time I checked they still owned the domain and it still leads to X.

they still own the domain, but it only redirects now. it is just `x.com` now (except for the help page for some reason)

**Comment by dreamscached** [comment]
Google owns many typesquatter domains, doesn't imply we can call it 'googel' or whatever.

**Comment by Booplicate** [comment]
@dreamscached the lilmonix is a different react and iirc it's disabled already

> they still own the domain, but it only redirects now

Which is why I said they didn't and won't get rid of it.

**Comment by dreamscached** [comment]
> @dreamscached the lilmonix is a different react and iirc it's disabled already

I know, just mentioned it as a point towards removing this react too. I don't see a real use for it, unlike Wikipedia or MAS Discord (or do we not have a react for that? I don't remember.)

**Comment by TheBearodactyl** [comment]
i just opened a PR that would change Twitter to bluesky

**Comment by meoowmeowman** [comment]
> I don't think Elon's family relations are relevant in the question. The website and the company were renamed to X, officially.

Oh please, do NOT tell me you´re supporting him? Do you like know what he´s done? On top of the sexual harassment and transphobia, he also used to have an alt account on twitter where he roleplayed as one of his kids (A 3 year old to be exact), and yes, it is sadly confirmed it was him. And on top of that the babyfur allegations.. yikes

**Comment by dreamscached** [comment]
I'm not a fan of Elon and I do not approve of him, but I'm also not going to pretend like X has not been rebranded. Like I said, personal feelings towards him are irrelevant in the question, I believe.
